clean-leather.lic

```diff
@@ -20,11 +20,11 @@ class CleanLeather
     ]
     args = parse_args(arg_definitions)
 
-    settings = get_settings
+    @settings = get_settings
```

Comment:
This does not need to be an instance var (`@`) since it is not used outside this function

Reply:
It depends where the ensure_copper_on_hand call resides. I can review.

Reply:
It depends where the ensure_copper_on_hand call resides. I can review.